src/features/orders/ordersHelpers.ts

```diff
@@ -203,7 +209,7 @@ export async function check(
   provider: ethers.providers.Web3Provider,
   isSwapWithWrap?: boolean
 ): Promise<AppError[]> {
-  const strings = await (
+  const [count, strings] = await (
```

Comment:
strings is now undefined so it breaks here

Reply:
Works on my end and the swapErc20 contractV4Abi ABI indicates uint256, bytes32[]:
```
name: "check",
outputs: [
  { internalType: "uint256", name: "", type: "uint256" },
  { internalType: "bytes32[]", name: "", type: "bytes32[]" },
],
```